packages/core/src/modules/basic-messages/repository/BasicMessageRecord.ts

```diff
@@ -43,6 +48,8 @@ export class BasicMessageRecord extends BaseRecord<DefaultBasicMessageTags, Cust
       this.connectionId = props.connectionId
       this._tags = props.tags ?? {}
       this.role = props.role
+      this.threadId = props.threadId
```

Comment:
Is there a problem in old exchanges not having a threadId?

Reply:
I don't think so, besides the fact that it will not be possible for query for them.